shared/constants/types/chat2/message.tsx

```diff
@@ -62,10 +62,28 @@ export type PathAndOutboxID = {
 
 // optional props here may never get set depending on the type
 type _MessageCommon = {
+  attachmentType?: AttachmentType
```

Comment:
its way easier for the root type to have these as undefined, else everywhere you have to check the type which adds a ton of noise. could expand this and remove those checks now